manager/ui/hawtio/plugins/api-manager/ts/rpc.ts

```diff
@@ -12,10 +12,29 @@ module ApimanRPC {
     export var ApimanSvcs = _module.factory('ApimanSvcs', ['$resource', 'Configuration',
         function($resource, Configuration) {
             var endpoint = Configuration.api.endpoint + '/:entityType/:secondaryType';
+

```

Comment:
Normally don't like to leave commented blocks of code, but in this case since we'll be making improvements in the future, it may be best to leave it in there for reference.